density/db/db.py

```diff
@@ -189,7 +189,7 @@ def get_oauth_code_for_uni(cursor, uni):
     else:
         # If the code DNE, create a new one and insert into the database.
         new_code = ''.join(random.choice(
-            string.ascii_uppercase + string.digits) for x in xrange(32))
+            string.ascii_uppercase + string.digits) for _x in xrange(32))
```

Comment:
Yeah, I'll take a look into what this code is actually being used for. I'm not super familiar with postgres, but I can certainly try and take a look into adding those things into the schema.